<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<HTML>
 <HEAD>
   <TITLE> [mod_python] assert error &quot;have_pysqlite &gt; 0&quot;
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20assert%20error%20%22have_pysqlite%20%3E%200%22&In-Reply-To=">
   <META NAME="robots" CONTENT="index,nofollow">
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="023267.html">
   <LINK REL="Next"  HREF="023269.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[mod_python] assert error &quot;have_pysqlite &gt; 0&quot;</H1>
<table border=0 width="100%"><tr><td valign="top">
    <B>ying lcs</B> 
    <A HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20assert%20error%20%22have_pysqlite%20%3E%200%22&In-Reply-To="
       TITLE="[mod_python] assert error &quot;have_pysqlite &gt; 0&quot;">yinglcs at gmail.com
       </A><BR>
    <I>Sat Mar 10 15:03:15 EST 2007</I>
    <P><UL>
        <LI>Previous message: <A HREF="023267.html">[mod_python] Error in running configure in RHEL4
</A></li>
        <LI>Next message: <A HREF="023269.html">[mod_python] When using PSP is request object persistent?
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#23268">[ date ]</a>
              <a href="thread.html#23268">[ thread ]</a>
              <a href="subject.html#23268">[ subject ]</a>
              <a href="author.html#23268">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Hi,

I am trying to setup Apache with Trac which uses mod_python. I get the
following error:
assert have_pysqlite &gt; 0

And I have verify this  via command line as well, that seem no problem.
# python
Python 2.3.4 (#1, Feb  2 2005, 11:44:49)
[GCC 3.4.3 20041212 (Red Hat 3.4.3-9.EL4)] on linux2
Type &quot;help&quot;, &quot;copyright&quot;, &quot;credits&quot; or &quot;license&quot; for more information.
&gt;&gt;&gt;<i> import trac.db.sqlite_backend
</I>&gt;&gt;&gt;<i> trac.db.sqlite_backend._ver
</I>(3, 3, 13)
&gt;&gt;&gt;<i> trac.db.sqlite_backend.have_pysqlite
</I>2
&gt;&gt;&gt;<i> trac.db.sqlite_backend.sqlite.version
</I>
'2.3.3'

But when i access the Trac via Apache (with mod_python), i get the
following error:

File &quot;/usr/lib/python2.3/site-packages/trac/db/pool.py&quot;, line 101, in
get_cnx
    cnx = self._connector.get_connection(**self._kwargs)

  File &quot;/usr/lib/python2.3/site-packages/trac/db/sqlite_backend.py&quot;,
line 113, in get_connection
    return SQLiteConnection(path, params)

  File &quot;/usr/lib/python2.3/site-packages/trac/db/sqlite_backend.py&quot;,
line 140, in __init__
    assert have_pysqlite &gt; 0
</PRE>




<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="023267.html">[mod_python] Error in running configure in RHEL4
</A></li>
	<LI>Next message: <A HREF="023269.html">[mod_python] When using PSP is request object persistent?
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#23268">[ date ]</a>
              <a href="thread.html#23268">[ thread ]</a>
              <a href="subject.html#23268">[ subject ]</a>
              <a href="author.html#23268">[ author ]</a>
         </LI>
       </UL>

</td>
<td align="right" valign="top">
<script type="text/javascript"><!--
google_ad_client = "pub-9718360309690383";
google_ad_width = 120;
google_ad_height = 600;
google_ad_format = "120x600_as";
google_color_border = "CCCCCC";
google_color_bg = "FFFFFF";
google_color_link = "000000";
google_color_url = "666666";
google_color_text = "333333";
//--></script>
<script type="text/javascript"
  src="DISABLEhttp://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</td>
</table>

<hr>
<a href="http://mailman.modpython.org/mailman/listinfo/mod_python">More information about the Mod_python
mailing list</a><br>
</body></html>
